src/enums/filing-codes.ts

```diff
@@ -17,5 +17,9 @@ export enum FilingCodes {
   INCORPORATION_ULC = 'BCINC',
   REGISTRATION_SP = 'FRREG',
   REGISTRATION_GP = 'FRREG',
+  RESTORATION_FULL = 'RESTF', // for BC/BEN/CC/ULC
+  RESTORATION_LIMITED = 'RESTL', // for BC/BEN/CC/ULC
+  RESTORATION_LTD_TO_FULL = 'RESXF', // for BC/BEN/CC/ULC
+  RESTORATION_LTD_EXTEND = 'RESXL', // for BC/BEN/CC/ULC
```

Comment:
According to this from Mihai:

![image](https://user-images.githubusercontent.com/10002889/210865280-cfd3b3c9-95ca-4e46-85ec-cbd9d339679f.png)